plugins/faustwp/includes/settings/callbacks.php

```diff
@@ -11,6 +11,26 @@
 	exit;
 }
 
+add_action( 'admin_notices', __NAMESPACE__ . '\\frontend_url_notice' );
+/**
+ * Callback for WordPress 'admin_notices' action.
+ *
+ * Show frontend_url missing error if it doesn't exist.
+ *
+ * @return void
+ */
+function frontend_url_notice() {
+	$frontend_url_setting = faustwp_get_setting( 'frontend_uri' );
+
+	if ( empty( $frontend_url_setting ) ) {
+		?>
+			<div class="notice notice-error is-dismissible" style="background-color: #FFEAE9;">
+				<p><?php esc_html_e( 'Front-end site URL is required to utilize url rewrites and previews. It is highly recommended that you update it below to avoid unexpected behavior. See the ', 'faustwp' ); ?><a href="https://faustjs.org/docs/getting-started" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Getting Started Documentation', 'faustwp' ); ?></a> <?php esc_html_e( 'for more details.', 'faustwp' ); ?></p>
+			</div>
```

Comment:
Using printf might improve readability for future translators.

```suggestion
			<div class="notice notice-error is-dismissible" style="background-color: #FFEAE9;">
				<p><?php esc_html_e( 'Front-end site URL is required to utilize url rewrites and previews.', 'faustwp' ); ?></p>
				<p><?php esc_html_e( 'It is highly recommended that you update it below to avoid unexpected behavior.', 'faustwp' ); ?></p>
				<p><?php
					printf(
						__( 'See the %1$s Getting Started Documentation%2$s for more details.', 'faustwp' ),
						'<a href="https://faustjs.org/docs/getting-started" target="_blank" rel="noopener noreferrer">',
						'</a>'
					);
				?></p>
			</div>
```

[More info on the topic](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#best-practices-for-writing-strings)

Reply:
The way GitHub renders my code suggestion is less than ideal :octocat: 

Here is how it might look in a much roomier text editor
![wpengine:faustjs:1196](https://user-images.githubusercontent.com/6676674/210439995-102193f2-4194-47e3-b548-c9a8674f5e73.png)

Reply:
Rendered in wp-admin, this suggestion vs the current PR
<img width="1611" alt="Screen Shot 2023-01-03 at 4 01 37 PM" src="https://user-images.githubusercontent.com/6676674/210442071-45f11ea8-d971-4b48-93bb-444c08213bc1.png">